docs/reference/commandline/set.md

```diff
@@ -0,0 +1,30 @@
+<!--[metadata]>
```

Comment:
@hqhq I think maybe more words. :-D How often do I say that?

---> https://gist.github.com/moxiegirl/cc09673bb8b0c1cfca99

<!--[metadata]>
+++
title = "set"
description = "The set command description and usage"
keywords = ["resources, set, dynamically"]
[menu.main]
parent = "smn_cli"
+++
<![end-metadata]-->

## set

    Usage: docker set [OPTIONS] CONTAINER [CONTAINER...]

    Sets container resource limits

      --help=false               Print usage
      --blkio-weight=0           Block IO weight (relative weight)
      --cpu-shares=0             CPU shares (relative weight)
      --cpu-period=0             Limit the CPU CFS (Completely Fair Scheduler) period
      --cpu-quota=0              Limit the CPU CFS (Completely Fair Scheduler) quota
      --cpuset-cpus=""           CPUs in which to allow execution (0-3, 0,1)
      --cpuset-mems=""           Memory nodes (MEMs) in which to allow execution (0-3, 0,1)
      -m, --memory=""            Memory limit
      --memory-reservation=""    Memory soft limit
      --memory-swap=""           Total memory (memory + swap), '-1' to disable swap
      --kernel-memory=""         Kernel memory limit: container must be stopped 

The `docker set` command dynamically sets container resources.  Use this command to prevent containers from consuming too many resources from their Docker host.  With a single command, you can place limits on a single container or on many. To specify more than one container, provide space-separated list of container names or IDs.

With the exception of the `--kernel-memory` value, you can specify these options on a running or a stopped container. You can only set  `--kernel-memory` on a stopped container. When you run `docker set` on stopped container, the next time you restart it, the container uses those values.